### PR TITLE
JSC-5168 : saramaprom adjustments

### PR DESCRIPTION
**Pull request by Tantalor93**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,33 @@
+export GO111MODULE := on
```
Comment (tsvoboda-wandera) [inline Makefile]: minor: afaik this is no longer necessary in new go versions (which I would target)

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+MAKEFLAGS += --no-print-directory
+
+prepare:
+	@echo "Downloading tools"
+	@cat tools.go | grep _ | cut -f2 -d " " | xargs -tI % sh -c "go install %"
+
+check: prepare
+	@echo "Running check"
+ifeq (, $(shell which golangci-lint))
+	curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(GOPATH)/bin v1.50.1
```
Comment (tsvoboda-wandera) [inline Makefile]: minor: what about bumping to the newer version?

**Review by tsvoboda-wandera — APPROVED** [review]

File: Makefile (hunk 1 above)
Comment (Tantalor93) [reply Makefile]: removed 👍

File: Makefile (hunk 2 above)
Comment (Tantalor93) [reply Makefile]: sure, done. Will add github actions to automatically run linting and tests in next PR

**Review by rschone — APPROVED** [review]

**Pull request merged**